magma/development/initialize-projects

```diff
@@ -12,6 +12,7 @@ if [ "${MAGMA_ENV}" == "test" ]; then
 PGPASSWORD=password psql -h magma_db -U developer -d magma_development -c "DROP DATABASE magma_test;"
 PGPASSWORD=password psql -h magma_db -U developer -d magma_development -c "CREATE DATABASE magma_test;"
 PGPASSWORD=password psql -h magma_db -U developer -d magma_development -c "GRANT ALL PRIVILEGES ON DATABASE magma_test TO developer;"
+PGPASSWORD=password psql -h magma_db -U developer -d magma_test -c "CREATE SCHEMA labors;"
```

Comment:
I seemed to need this locally to run tests in the Magma container...so just adding it to the script for convenience.

Reply:
So, the way it works now is that it scans the config.yml for the test section and initializes schemas for projects  based on entries in that config.  Can you verify if that is happening for you locally?  In the same file, the for loop.

Reply:
Oh you know, I think this doesn't work for test, only development at the moment.  Maybe we could change the for loop to do a similar initialization for test section?

Reply:
It doesn't happen when I run `make -C magma shell`:

```
+ for hook in /app/development/*
+ '[' -x /app/development/initialize-projects ']'
+ /app/development/initialize-projects
DROP DATABASE
CREATE DATABASE
GRANT
+ '[' -n '' ']'
+ mkdir -p /app/data/uploads
+ mkdir -p /app/data/data_blocks
+ exec bash

 2021-12-03 19:15:06 ⌚  0a93cbb12867 in /app
○ → bundle exec rspec spec/query_spec.rb 

An error occurred while loading spec_helper.
Failure/Error: Magma.instance.db[:attributes].truncate

Sequel::DatabaseError:
  PG::UndefinedTable: ERROR:  relation "attributes" does not exist
```

Perhaps the $UPDATE_STATE environment variable isn't getting set correctly? Would a better fix be to add that variable to Magma's `docker-compose.yml`, like `    'UPDATE_STATE': '1'` under the app?

Reply:
If I update Magma's `docker-compose.yml`, I get the expected output:

```
+ for hook in /app/development/*
+ '[' -x /app/development/initialize-projects ']'
+ /app/development/initialize-projects
DROP DATABASE
CREATE DATABASE
GRANT
Your Gemfile lists the gem curb (>= 0) more than once.
You should probably keep only one of them.
Remove any duplicate entries and specify the gem only once.
While it's not a problem now, it could cause errors if you change the version of one of them later.
Migrating to latest, globally
Initializing ./spec/labors/...
```